packages/subgraph/src/plugin/adminMembers.ts

```diff
@@ -0,0 +1,79 @@
+import {Administrator, AdministratorAdminPlugin} from '../../generated/schema';
+import {
+  Granted,
+  Revoked,
+} from '../../generated/templates/AdminMembers/PermissionManager';
+import {generateAdministratorAdminPluginEntityId} from '../utils/ids';
+import {
+  generateEntityIdFromAddress,
+  generatePluginEntityId,
+} from '@aragon/osx-commons-subgraph';
+import {dataSource, store} from '@graphprotocol/graph-ts';
+
+export function handleGranted(event: Granted): void {
+  if (
+    isCorrectEvent(
+      event.params.permissionId.toHexString(),
+      event.params.where.toHexString()
+    )
+  ) {
+    const pluginAddress = event.params.where;
+    const administratorAddress = event.params.who;
+    const pluginEntityId = generatePluginEntityId(pluginAddress);
+    const administratorEntityId =
+      generateEntityIdFromAddress(administratorAddress);
+    let administrator = Administrator.load(administratorEntityId);
+    if (!administrator) {
+      administrator = new Administrator(administratorEntityId);
+      administrator.address = administratorEntityId;
+      administrator.save();
+    }
+
+    const administratorMappingId = generateAdministratorAdminPluginEntityId(
+      pluginAddress,
+      administratorAddress
+    );
+    let administratorPluginMapping = AdministratorAdminPlugin.load(
+      administratorMappingId
+    );
+    if (!administratorPluginMapping) {
+      administratorPluginMapping = new AdministratorAdminPlugin(
+        administratorMappingId
+      );
+      administratorPluginMapping.administrator = administratorEntityId;
+      administratorPluginMapping.plugin = pluginEntityId;
+      administratorPluginMapping.save();
+    }
+  }
+}
+
+export function handleRevoked(event: Revoked): void {
+  if (
+    isCorrectEvent(
+      event.params.permissionId.toHexString(),
+      event.params.where.toHexString()
+    )
+  ) {
+    // where is the plugin address
```

Comment:
if you're going to write these as comments, better to just use local variables for readability